### ISSUE TITLE
Images not loading correctly in Firefox for Android when Photon is enabled

### ISSUE THREAD

**Issue by SiobhyB** [Automattic/jetpack#3903]
Images are not loading correctly when viewing http://www.strange-works.de in Firefox for Android while Photon is enabled on that site.

#### What I expected

I expected all of the images on the home page of http://www.strange-works.de to load correctly, as they do when viewing that site on Chrome for Android. Here's an example:

![screenshot_20160523-1635482](https://cloud.githubusercontent.com/assets/2998162/15476157/7e59f01a-2105-11e6-8626-2405c42e593b.png)

#### What happened instead

The images didn't load correctly while viewing the site in Firefox for Android:

![screenshot_20160523-164032](https://cloud.githubusercontent.com/assets/2998162/15476136/634446f4-2105-11e6-8c04-f952278c6cfc.png)

The images do load correctly when Photon is disabled.

#### Steps to reproduce the issue

Visit http://www.strange-works.de in Firefox for Android while Photon is enabled.

The site has the Pictorico theme active.

Reported in 2660697-t and here:

https://wordpress.org/support/topic/firefox-mobile-issue-using-jetpacks-photon-function?replies=5

Similar report:

https://wordpress.org/support/topic/photon-in-firefox-molile-gives-random-image-sizes?replies=7

**Comment by kraftbj** [comment]
@SiobhyB Do we know if it is isolated to that theme?

**Comment by SiobhyB** [comment]
@kraftbj: The site's owner isn't able to switch themes as it's a live site. I'm going to test on my own public test site, though, and will update this thread.

**Comment by SiobhyB** [comment]
I was able to replicate on my own test site.

My test site's homepage in **Firefox for Android** with **the Pictorico theme** and **Photon enabled**. The images load at the expected size:

https://cloudup.com/cOWbw0h8a3j

My test site's homepage in **Firefox for Android** with **the Pictorico theme** and **Photon disabled**. The images all appear to load at random size and don't take up the full width of the available space:

https://cloudup.com/cMBPj9B_3zv

My test site's homepage in **Firefox for Android** with **the Twenty Sixteen theme** and **Photon enabled**. The images load at the expected size:

https://cloudup.com/cRbJWKczzYS

The issue can only be replicated with Pictorico while Photon is enabled. Are there any known issues with Photon that would be helpful when taking this issue to the theme's developers?

**Comment by jeherve** [comment]
> Are there any known issues with Photon that would be helpful when taking this issue to the theme's developers?

Not really, but I wonder if this could be caused by the changes Photon makes to the image markup (especially the image dimensions). You might be able to fix this in the theme with CSS.

Related: http://stackoverflow.com/q/14718033/1045686

**Comment by SiobhyB** [comment]
Thank you for the information! I created a ticket in 3934-wpcom-themes.

**Comment by jeherve** [comment]
Closing this for now because of the lack of activity on this. We can always reopen in the future if needed.
